src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -239,6 +239,31 @@ public void StartListening()
         /// Invoke a method on the server.
         /// </summary>
         /// <param name="targetName">The name of the method to invoke on the server. Must not be null or empty string.</param>
+        /// <param name="argument">Method argument, must be serializable to JSON.</param>
+        /// <returns>A task that completes when the server method executes.</returns>
+        /// <exception cref="OperationCanceledException">
+        /// Result task fails with this exception if the communication channel ends before the server indicates completion of the method.
+        /// </exception>
+        /// <exception cref="RemoteInvocationException">
+        /// Result task fails with this exception if the server method throws an exception.
+        /// </exception>
+        /// <exception cref="RemoteMethodNotFoundException">
+        /// Result task fails with this exception if the <paramref name="targetName"/> method is not found on the target object on the server.
+        /// </exception>
+        /// <exception cref="RemoteTargetNotSetException">
+        /// Result task fails with this exception if the server has no target object.
+        /// </exception>
+        /// <exception cref="ArgumentNullException">If <paramref name="targetName"/> is null.</exception>
+        /// <exception cref="ObjectDisposedException">If this instance of <see cref="JsonRpc"/> has been disposed.</exception>
+        public Task InvokeAsync(string targetName, object argument)
+        {
+            return this.InvokeAsync<object>(targetName, argument);
```

Comment:
Thanks for adding this overload. However this one may be missing a test, as it would throw `StackOverflowException`. You'll need to explicitly surround `argument` with `new object[] { argument }` in order to force the compiler to call the other overload.